docs/Data Model.md

```diff
@@ -62,7 +62,7 @@ Owned by:
 SHOULD change if:
 
 - Device ID changes
-- A different physical interface (such as an SDI output) is mapped to consume the essence
+- A different physical interface (such as an SDI output) is mapped to consume the stream
```

Comment:
Why "stream" here and "signal" above?

Reply:
Ah good point, it's not the stream that was received at the Receiver, it's the signal that the Output is, hmm, consuming seems wrong, as does originating on the Input side?

Reply:
My bad, here should be "signal".
```suggestion
- A different physical interface (such as an SDI output) is mapped to consume the signal
```
It is a different signal, but should we emphasize this? If we allow Outputs not associated with Receivers, I don't see how we can define what is the signal that the Output consumes. It may be any signal from anywhere, isn't it?